src/scripts/main.js

```diff
@@ -1,7 +1,93 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Write your code here
+const button = document.querySelector('.button');
+const score = document.querySelector('.game-score');
+
+const rows = document.querySelectorAll('tr');
```

Comment:
It's best practice to verify that the expected number of 'tr' elements are selected to avoid potential errors when trying to access cells in the 'drawBoard' function.